### ISSUE TITLE
bug: Long Press Gesture on Phone Link Does Not Work in Safari on iOS in Ionic Application

### ISSUE THREAD

**Issue by bryplano** [ionic-team/ionic-framework#20553]
# Bug Report

## Ionic version:

[x] **5.x and 4.x**

## Current behavior

In Safari on iOS, when long pressing on a telephone link like this one in an Ionic application:

```
<a href="tel:123-456-7890">123-456-7890</a>
```
nothing happens. The tap gesture is still recognized properly, but a long press does nothing.

**NOTE**: Cordova or Capacitor is NOT REQUIRED to reproduce this behavior! This happens in a standalone Ionic application. 

## Expected behavior

When long pressing on a telephone number link in Safari on iOS, it is expected that a context menu will appear (similar to a tap, but with more options) providing the ability for the user to use other applications with the phone link.

![File (2)](https://user-images.githubusercontent.com/39273819/74842970-d3f68a80-52f0-11ea-948d-c1782d89489b.jpg)

## Reproducing the expected behavior

You can reproduce the expected behavior by going to this [sample Angular application](https://kws-playground.firebaseapp.com/) and long pressing on the "Call me" link in Safari on iOS. This test was performed by @kensodemann to confirm that the problem was not an Angular issue.

## Steps to reproduce:

**Note**: This application is being hosted on Firebase for ease of access - the repo can be found here: https://github.com/bryplano/TelAndMap

1. On an iPhone, in Safari, navigate to https://test-fc17f.firebaseapp.com/, which is an Ionic application hosted on Firebase
2. Long press on either of the links at the top of the screen
3. Note that nothing happens
4. You can tap on the links to see the phone number action sheet pop up, but the long press gesture does nothing

## Related code

Sample app : https://github.com/bryplano/TelAndMap

Specific code line: https://github.com/bryplano/TelAndMap/blob/master/src/app/home/home.page.html#L10

## `ionic info`

```
Ionic:

   Ionic CLI                     : 6.1.0 
   Ionic Framework               : @ionic/angular 4.11.10
   @angular-devkit/build-angular : 0.801.3
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.1.2

Cordova:

   Cordova CLI       : 9.0.3 (cordova-lib@9.0.2)
   Cordova Platforms : ios 5.1.1
   Cordova Plugins   : cordova-plugin-ionic-webview 4.1.3, (and 6 other plugins)

Utility:

   cordova-res : 0.9.0
   native-run  : not installed

System:

   ios-deploy : 1.10.0
   ios-sim    : 8.0.2
   NodeJS     : v10.19.0
   npm        : 6.13.4
   OS         : macOS Catalina
   Xcode      : Xcode 11.3.1 Build version 11C504
```

**Comment by kensodemann** [comment]
The app @bryplano created is v4.11.10, but I also tried this on a similar v5 app with the same results (just to confirm that this is still an issue in v5).

**Comment by NikolaPeevski** [comment]
@bryplano Can you commit your changes regarding the cordova plugins ? Trying to install all npm packages and doing ionic info shows no cordova plugins installed

**Comment by kensodemann** [comment]
Adding the following to the `global.scss` file worked in my test app without negative consequences:

```css
* {
  -webkit-touch-callout: default;
}
```

**Comment by liamdebeasi** [comment]
Hi everyone,

I took a look at this, and this is by design. We set `-webkit-touch-callout: none`: https://github.com/ionic-team/ionic-framework/blob/4ccc150edff4e3f33f453343aedb081d920b2f90/core/src/css/structure.scss#L14C8-L14C10

Linking to a page (even within an application) causes an `a` element to render. However, linking within an app typically does not have a long press gesture associated with it. This behavior is usually reserved for external links (such as tapping a phone number).

Developers should set `-webkit-touch-callout: default` on specific links where they do want this behavior to be activated as noted in https://github.com/ionic-team/ionic-framework/issues/20553#issuecomment-593448943.
